config/settings.yaml.example

```diff
@@ -37,6 +37,11 @@
 #:logging:
 #  :level: debug
 
+# It is also possible to override settings per environment:
+#:logging:
+#  :test:
+#    :pattern: "[%c|%.1l] %m\n"
```

Comment:
What's exactly isn't clear? What you suggest does not change a thing, I want to configure SYSLOG output and output format for particular environment. We already have the ability to override globally, this adds an option to add an environment. The old behavior still works fine.